.github/workflows/pr.yaml

```diff
@@ -0,0 +1,40 @@
+name: pr
+
+on:
+  push:
+    branches:
+      - "pull-request/[0-9]+"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  pr-builder:
+    needs:
+      - wheel-build-ucx-py
+      - wheel-tests-ucx-py
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/pr-builder.yaml@main
+  wheel-build-ucx-py:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@upkeep/manylinux-improvements
+    with:
+      build_type: pull-request
+      package-name: ucx_py
+      package-dir: .
+      cibw-before-build: "cp {package}/.github/build-scripts/patch-wheel.sh /patch-wheel.sh && sed -i 's/name = \"ucx-py\"/name = \"ucx-py-cu11\"/g'  {package}/pyproject.toml"
+      post-repair-amd64: "/patch-wheel.sh {dest_dir}/*.whl"
+      post-repair-arm64: "/patch-wheel.sh {dest_dir}/*.whl"
+  wheel-tests-ucx-py:
+    needs: wheel-build-ucx-py
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-test.yml@upkeep/manylinux-improvements
+    with:
+
+      build_type: pull-request
+      package-name: ucx_py
+      test-docker-options: "--cap-add CAP_SYS_PTRACE --shm-size=8g"
+      test-before-arm64: "pip install cupy-cuda11x -f https://pip.cupy.dev/aarch64"
+      test-unittest: "pytest -k 'not test_send_recv_am' --cache-clear -vs ./ucp/_libs/tests/ && pytest --cache-clear -vs ./tests"
```

Comment:
Let's link to https://github.com/rapidsai/ucx-py/issues/797 here and in test.yaml so we know why we're skipping this test and when we can remove the filter.